webapp-django/crashstats/topcrashers/tests/test_views.py

```diff
@@ -0,0 +1,223 @@
+import mock
+import pyquery
+from nose.tools import eq_, ok_
+
+from django.core.urlresolvers import reverse
+
+from waffle.models import Switch
+
+from crashstats.crashstats.tests.test_views import (
+    BaseTestViews, Response, mocked_post_123
+)
+from crashstats.supersearch.tests.common import (
+    SUPERSEARCH_FIELDS_MOCKED_RESULTS,
+)
+
+
+class TestViews(BaseTestViews):
+    base_url = reverse('topcrashers:topcrashers')
+
+    @staticmethod
+    def setUpClass():
+        TestViews.switch = Switch.objects.create(
```

Comment:
Kill kill kill all waffle switches that aren't useful. 
Waffle flags are useful when you're swapping one feature for another. But in this case, it's a new URL. And it's not a secret. Waffle is a pain to deal with locally and it's annoying that you have manually mess around with django commands on the servers. 

I basically think we should delete almost ALL waffle switches everywhere. Perhaps you know better examples, but I can't think of a single one that is useful to us. 

Also, note that with waffle switches the view can't begin until Django has talked to the postgres database so they're often a performance penalty.